index.html

```diff
@@ -3088,21 +3088,19 @@ <h2 id="OfflineAudioCompletionEvent">
           </dl>
         </section>
         <section>
-          <h3 id="lifetime-AudioNode" class="informative">
+          <h3 id="lifetime-AudioNode">
             Lifetime
           </h3>
           <p>
-            <i>This section is informative.</i>
+            The following behaviors provide a normative description of the
+            conditions under which an AudioNode is alive, meaning that it MUST
+            be retained in the graph by an implementation. Where these
+            conditions do not apply, AudioNodes MAY be released by an
+            implementation.
           </p>
           <p>
-            An implementation may choose any method to avoid unnecessary
-            resource usage and unbounded memory growth of unused/finished
-            nodes. The following is a description to help guide the general
-            expectation of how node lifetime would be managed.
-          </p>
-          <p>
-            An <a><code>AudioNode</code></a> will live as long as there are any
-            references to it. There are several types of references:
+            An <a><code>AudioNode</code></a> must remain alive as long as there
```

Comment:
Is this a MUST in the sense of RFC2119, or a must in english ? In any case, this behaviour is well defined on the web platform already, no need to define it here again. We can simply say that we have self-references in some cases, and everything will work out fine.